src/declarations/stencil-public-compiler.ts

```diff
@@ -1845,6 +1845,27 @@ export interface OutputTargetDist extends OutputTargetBase {
   dir?: string;
 
   collectionDir?: string | null;
+  /**
+   * When `true` this flag will transform aliased import paths defined in
+   * a project's `tsconfig.json` to relative import paths in the compiled output's
+   * `dist-collection` bundle if it is generated (i.e. `collectionDir` is set).
+   *
+   * Paths will be left in aliased format if `false` or `undefined`.
+   *
+   * // tsconfig.json
+   * {
```

Comment:
Can we wrap this in a code block to make sure that formatting is adhered to in renderers for JSDoc?

Sometimes, editors like VSCode will not respect newlines like so:
![Screen Shot 2022-08-30 at 4 02 01 PM](https://user-images.githubusercontent.com/1930213/187532154-ab61193e-0d26-4240-b4fe-2a0dc39bc01c.png)

Reply:
Leverage the JSDoc `@example` tag. That preserves standard code formatting and syntax highlighting. Up in [646bc5c](https://github.com/ionic-team/stencil/pull/3559/commits/646bc5c5906e74d45ad5d69ec137f05624764deb)